UpdaterUpdater/src/at/chaosfield/updaterUpdater/UpdaterUpdater.java

```diff
@@ -31,6 +31,15 @@ public UpdaterUpdater(String[] args){
 
         String version = null;
 
+        try {
+            String directoryName = args[2].concat(this.getClassName());
```

Comment:
Why do you check for a folder called "UpdaterUpdater" (the class name) and why would you check if the mods folder exists inside of the packupdate updater and not inside of packupdate itself?